src/themes/default/components/core/blocks/Wishlist/Product.vue

```diff
@@ -13,6 +13,20 @@
         <div class="h6 cl-bg-tertiary pt5 sku">
           {{ product.sku }}
         </div>
+        <div v-if="canAddToCart()">
```

Comment:
computed property as well
```suggestion
        <div v-if="canAddToCart">
```